internal/cmd/flags.go

```diff
@@ -77,12 +78,22 @@ var (
 		Shorthand: "m",
 	}
 
-	PflagUserMetadataSignUsage = "{key}={value} pairs that are added to the signature payload"
+	PflagUserMetadataSignUsage   = "{key}={value} pairs that are added to the signature payload"
```

Comment:
nit extra space

Reply:
this one was added by gofmt to align w/ the line below i believe